src/Auth/index.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router';
+import { connect } from 'react-redux';
+
+const Auth = App => NotAuthorized =>
+	class extends Component {
+		state = {
+			loggedIn: null,
+		};
+
+		componentDidMount() {
+			if (localStorage.getItem('user')) {
```

Comment:
Make sure this eventually checks the login against the backend database of users, not just localStorage.